lms/djangoapps/teams/static/teams/templates/team_count_detail.underscore

```diff
@@ -0,0 +1 @@
+<%- interpolate('%(team_count)s Teams', {team_count: team_count}, true) %>
```

Comment:
I find this very odd. The template produces exactly the result of using ``interpolate``, so why make this a template at all?